launch/launch/frontend/entity.py

```diff
@@ -82,3 +82,12 @@ def get_attr(
             Only happens in frontend implementations that do type coercion.
         """
         raise NotImplementedError()
+
+    def assert_entity_complely_parsed(self):
```

Comment:
@ivanpauno did you mean:
```suggestion
    def assert_entity_completely_parsed(self):
```
?

Reply:
ups, fixed in https://github.com/ros2/launch/pull/468/commits/dc8fc265f5ef4c7be2645210621d969d2dd0d7f8